security/admin.py

```
@@ -0,0 +1,13 @@
+from django.contrib import admin
+from .models import AuditLog, Role
+
+@admin.register(AuditLog)
+class AuditLogAdmin(admin.ModelAdmin):
+    list_display = ['id', 'user_id', 'path', 'method', 'status_code', 'created_at']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)